src/main/java/org/javarosa/entities/EntityFormPostProcessor.java

```diff
@@ -39,4 +40,25 @@ public void processForm(FormEntryModel formEntryModel) {
             formEntryModel.putAttachment(new EntitiesAttachment(emptyList()));
         }
     }
+
+    private boolean createIsRelevant(FormInstance mainInstance) {
+        TreeElement root = mainInstance.getRoot();
+        List<TreeElement> meta = root.getChildrenWithName("meta");
+        if (!meta.isEmpty()) {
+            List<TreeElement> entity = meta.get(0).getChildrenWithName("entity")
+                .stream()
+                .filter(node -> node.getNamespace().equals("http://www.opendatakit.org/xforms/entities"))
+                .collect(Collectors.toList());
+
+            if (!entity.isEmpty()) {
+                List<TreeElement> create = entity.get(0).getChildrenWithName("create");
+
+                if (!create.isEmpty()) {
+                    return create.get(0).isRelevant();
```

Comment:
@lognaturel from our discussion [in the spec](https://docs.google.com/document/d/1-O_wT0JZZFaWvzHl_krrzERJcnrdGMKP55ZZ_hxV0gU/edit#), it seems like we actually don't want this check yet as we won't be included it (support for relevance on `create`) in the spec. Is that right? I think it'll be confusing to include if it's not something we document or that Central supports.

Reply:
I think the important thing is to check for the existence of the `create` node *in the instance to be submitted*. I was previously thinking that it would exist if relevant and not exist if non-relevant but now I realize that may not be true depending on when the pruning of non-relevant nodes happens.

Reply:
> Central is only looking for create on submission so it will implicitly support conditional creation right away.

Ah ok. I think we should just include it in the spec then so it's clear that's happening and so Central/Collect are both consistent to what you've documented.

> previously thinking that it would exist if relevant and not exist if non-relevant but now I realize that may not be true depending on when the pruning of non-relevant nodes happens

It looks like we don't "remove" the non-relevant nodes until serialization: https://github.com/getodk/javarosa/blob/1b4a7b42ee599f32eaad3fe9118178d009782dc6/src/main/java/org/javarosa/model/xform/XFormSerializingVisitor.java#L179.

Given that, I think we do this check to be able to expose entities at "runtime".

Reply:
> I think we should just include it in the spec

You mean make conditional create part of the initial spec, right? That sounds good to me!

Reply:
> You mean make conditional create part of the initial spec, right?

Yup!